api/rpc/project/project.proto

```diff
@@ -2,15 +2,30 @@ syntax = "proto3";
 
 package project;
 
-service Project {
-  rpc Create (CreateRequest) returns (CreateReply) {}
+service ProjectSvc {
+  rpc Create (Project) returns (ProjectReply) {}
```

Comment:
In order to follow the RPC style, I would create a "CreateRequest" that includes a "Project" instead of directly passing the project as input parameter. In this case, you can update the creation request without breaking compatibility (and just add new fields).
The same applies for other rpc calls.